ReactWindows/Playground/AppReactPage.cs

```diff
@@ -11,15 +11,15 @@ public override string MainComponentName
         {
             get
             {
-                return "UIExplorerApp";
+                return "Playground";
             }
         }
 
         public override string JavaScriptMainModuleName
         {
             get
             {
-                return "Examples/UIExplorer/UIExplorerApp.windows";
+                return "ReactWindows/Playground/index.windows";
```

Comment:
It was accidentally changed before to UIExplorer.  UIExplorer may have different Package.appxmanifest requirements (e.g., location permissions) that we don't want to put in the basic playground app.

---
In reply to: [70991300](https://github.com/ReactWindows/react-native-windows/pull/526#discussion_r70991300) [](ancestors = 70991300)